src/pages/msme/ViewCreditOptions.tsx

```diff
@@ -200,6 +204,23 @@ function ViewCreditOptions() {
                 options={SECTOR_TYPES}
                 placeholder={t('Sector')}
               />
+              <FormInput
+                className="w-3/5"
```

Comment:
I assume these form inputs are stacked? I was just checking if w-3/5 made sense or needed to be updated: https://tailwindcss.com/docs/width#percentage-widths

Reply:
Yes, they are, for reference:
![image](https://github.com/user-attachments/assets/50a9cee1-4116-41f2-94dd-4d262ea3297c)

Reply:
Actually, I think COP is enough as a place holder, I removed the 0 part